raaghu-elements/src/rds-stepper/rds-stepper.tsx

```
@@ -1,405 +1,451 @@
+  advanceList?: RdsStepperAdvanceListProps[];
 }
 
 const RdsStepper = (props: RdsStepperProps) => {
-    const [page, setPage] = useState(1);
-    const [checkedStates, setCheckedStates] = useState<boolean[]>(Array(props.stepperDetails?.length).fill(false));
-    const formTitles = [
-        { stepName: "Step 1", stepabname: "Profile" },
-        { stepName: "Step 2", stepabname: "Positions" },
-        { stepName: "Step 3", stepabname: "Settings" },
-    ];
+  const [page, setPage] = useState(1);
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)